gems/pending/util/system/evm_watchdog.rb

```diff
@@ -1,7 +1,11 @@
 require 'time'
+require 'linux_admin'
+require 'fileutils'
 
 module EvmWatchdog
   PID_LOCATION = '/var/www/miq/vmdb/tmp/pids/evm.pid'.freeze
+  UPDATE_FILE  = '/var/www/miq/vmdb/tmp/miq_update'.freeze
```

Comment:
@carbonin Sidenote, next/future PR:  It feels like the tmp directory could be provided if this was a class in the initialize, then the calling code could be concerned with the tmp directory and this code could be agnostic to pathing.